Updated frameworks and dependencies

FrendsSarlinS opened the pull request:
Jira ticket on Frends Support: FSUP-10058

FrendsSarlinS committed:
Updated frameworks and dependencies

OssiGalkin commented:
Is there any other reason say this is a breaking change than that previous library update said it was braking change? 

I cant see why something would necessary brake from frends perspective.

OssiGalkin commented:
If there is no braking change new version could be 3.1.0

FrendsSarlinS committed:
Version number changed to next update

FrendsSarlinS committed:
Version number change also to csproj

FrendsSarlinS commented:
> Is there any other reason say this is a breaking change than that previous library update said it was braking change?
> 
> I cant see why something would necessary brake from frends perspective.

That's actually correct, nothing broke after that change so I removed that part from release notes and changed version number to 3.1.0 as suggested.
